stdlib/mlang/symbolize.mc

```diff
@@ -0,0 +1,787 @@
+include "bool.mc"
+include "error.mc"
+include "map.mc"
+include "name.mc"
+include "set.mc"
+
+include "./pprint.mc"
+include "./ast.mc"
+include "./language-composer.mc"
+
+include "mexpr/symbolize.mc"
+include "mexpr/ast-builder.mc"
+
+let fst : all a. all b. (a, b) -> a = lam p.
+    match p with (res, _) in res
+
+let snd : all a. all b. (a, b) -> b = lam p.
+    match p with (_, res) in res
+
+
+let mapDisjoint : all a. all b. (Map String a) -> (Map String b) -> Bool = lam l. lam r.
+    let leftKeys = setOfSeq cmpString (mapKeys l) in 
```

Comment:
`setOfKeys` exists

Reply:
Also, this function is probably more efficiently implemented with `mapIntersectWith`, to not create more intermediate values.